### DISCUSSION TITLE
How to upgrade a blitz app

### DISCUSSION THREAD

**Discussion by saleebm** [blitz-js/blitz#1672] [General] [answered]
Hello, thanks everyone for this awesome architecture. I am wondering how to upgrade my blitz up to the latest version. I see the `blitz new` does set to exact versions in the package file. There are also no docs that I found clarifying the process to upgrade. Would be interested in how others upgrade to the latest blitz version? I just want to make sure my dependencies match up. Thank you all.

**Comment by flybayer** [comment, marked as answer]
Hey! Here you go:

1. Upgrade installed blitz version via npm or yarn
2. Check Blitz release notes for any breaking changes that you need to fix

The other dependencies can be updated just like normal. They don't depend on the blitz version with the exception of Prisma.

To use Prisma 2.13+, you will use the prisma CLI directly instead of the `blitz db migrate` command. The `blitz db migrate` command is going to be removed.
